devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy

```diff
@@ -114,8 +114,9 @@ def updated() {
 def configure() {
 	commands([
 		zwave.manufacturerSpecificV2.manufacturerSpecificGet(),
-		zwave.batteryV1.batteryGet()
-	], 6000)
+		zwave.batteryV1.batteryGet(),
+		zwave.sensorBinaryV2.sensorBinaryGet(sensorType: zwave.sensorBinaryV2.SENSOR_TYPE_DOOR_WINDOW)
```

Comment:
We might send the wakeUpNoMoreInformation before this gets to the device.

Send sensorBinaryGet first, then manufacturerSpecificGet, and take batteryGet out of this. It will be sent from the ManufacturerSpecificReport handler.

It's not safe to send so many Get commands back to back in a generic DTH because they can back up and collide with some devices.